### ISSUE TITLE
Set `--save` and `--preserve` flags by default

### ISSUE THREAD

**Issue by Masqueey** [AloneLiberty/FlipperNestedRecovery#8]
In my case the process was interrupted and thereby lost any progress made. The cracked keys were added to my library, but before I could continue cracking, I had to rescan the entire card again. Why are these not set by default (or set to save the progress if interrupted)? Could these be set by default?

**Comment by AloneLiberty** [comment]
You interrupted by yourself? I don't remember if it will remove nonces unless full recovery is competed. Will check.
--save and --preserve are debug flags and won't be enabled by default.

**Comment by Masqueey** [comment]
Yes, just ctrl+C'd the process, but expected to be able to continue from there. Maybe that is intended behavior though.

**Comment by AloneLiberty** [comment]
Checked code, not intended. Will fix, won't remove nonces if user quits app
